configuration.cpp

```diff
@@ -15,11 +15,21 @@ namespace irods {
                     }
                 }; // capture_parameter
 
+                std::string Label;
+                try {
+                    // integer-or-string parameters
+                    job_limit = LABEL("job_limit_per_collection_indexing_operation",cfg,int);
+                    test_debug = LABEL("test_debug",cfg,int);
```

Comment:
keeping this?

Reply:
Yes, that little construction is a way of letting the user know which parameter is acting up in case of an exception.
I'm sure it's considered sub-optimal in some circles, but I could not come up with something better....@korydraughn?

Reply:
Oh I see, the test_debug.  Yes, it needs to stay there because of the new test for issue 91

Reply:
okay